docs/outlook/debug-autolaunch.md

```diff
@@ -0,0 +1,136 @@
+
+1. Back in the command line window, open VS Code.
+
+    ```command&nbsp;line
+    code .
+    ```
+
+1. In VS Code, open the file **./.vscode/launch.json** and add the following excerpt to your list of configurations. Save your changes.
+
+    ```json
+    {
+      "name": "Direct Debugging",
+      "type": "node",
+      "request": "attach",
+      "port": 9229,
+      "protocol": "inspector",
+      "timeout": 600000,
+      "trace": true
+    }
+    ```
+
+### Other
+
+1. Create a new folder called **Debugging** (perhaps in your **Desktop** folder).
+1. Open VS Code.
+1. Go to **File** > **Open Folder**, navigate to the folder you just created, then choose **Select Folder**.
+1. Select the **Debug** item in the left menu.
+
+    ![Screenshot of Debug icon in left menu of VS Code](../images/vs-code-debug.png)
+
+1. Select the **create a launch.json file** link.
+
+    ![Screenshot of link to create a launch.json file in VS Code](../images/vs-code-create-launch.json.png)
+
+1. In the menu, select **Edge: Launch** to create a launch.json file.
+1. Add the following excerpt to your list of configurations. Save your changes.
+
+    ```json
+    {
+      "name": "Direct Debugging",
+      "type": "node",
+      "request": "attach",
+      "port": 9229,
+      "protocol": "inspector",
+      "timeout": 600000,
+      "trace": true
+    }
+    ```
+
+## Attach VS Code
+
+1. To find the add-in's **bundle.js**, open the following folder in Windows Explorer and search for your add-in's **Id** (found in the manifest).
+
+    ```text
+    %LOCALAPPDATA%\Microsoft\Office\16.0\Wef
+    ```
+
+    Open the folder prefixed with this ID and copy its full path. In VS Code, open **bundle.js** from that folder. The pattern of the file path should be as follows:
+
+    `%LOCALAPPDATA%\Microsoft\Office\16.0\Wef\{[Outlook profile GUID]}\[encoding]\Javascript\[Add-in ID]_[Add-in Version]_[locale]\bundle.js`
+
+1. Place breakpoints in bundle.js where you want the debugger to stop.
+1. In the **DEBUG** dropdown, select the name **Direct Debugging** then select the **Run** button.
+
+    ![Screenshot of selecting Direct Debugging from configuration options in the VS Code Debug dropdown](../images/outlook-win-autolaunch-debug-vsc.png)
+
+## Debug
+
+1. After confirming that the debugger is attached, return to Outlook then choose the **OK** button in the **Debug Event-based handler** dialog.
+
+1. Your breakpoints in VS Code should now be hit, enabling you to debug your event-based activation code.
+
+## Stop debugging
+
+To stop debugging for:
+
+- The rest of the current Outlook desktop session, choose the **Cancel** button in the **Debug Event-based handler** dialog. To re-enable debugging, restart Outlook desktop.
+- Subsequent Outlook sessions and prevent the **Debug Event-based handler** dialog from popping up, delete the associated registry key or set its value to 0: `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\Wef\Developer\[Add-in ID]\UseDirectDebugger`.
```

Comment:
```suggestion
- To stop debugging for subsequent Outlook sessions and prevent the **Debug Event-based handler** dialog from popping up, delete the associated registry key or set its value to 0: `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\Wef\Developer\[Add-in ID]\UseDirectDebugger`.
```

Reply:
What about this?
```suggestion

To prevent the **Debug Event-based handler** dialog from popping up and stop debugging for subsequent Outlook sessions, delete the associated registry key or set its value to `0`: `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\Wef\Developer\[Add-in ID]\UseDirectDebugger`.
```

Reply:
Looks good to me!